src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <main class="container">
+      <div class="circle"></div>
+      <div class="hours"></div>
+      <div class="minutes"></div>
+      <div class="seconds"></div>
```

Comment:
```suggestion
    <div class="watch">
      <div class="circle"></div>
      <div class="hours"></div>
      <div class="minutes"></div>
      <div class="seconds"></div>
    <div>
```
make container for all parts of watch and set width and height 80 vmin for this container instead main
now wathc not full visible on screen
![Screenshot from 2023-06-02 12-16-36](https://github.com/mate-academy/layout_stop-watch/assets/33029864/990dc5c7-73db-450c-aa2e-376db5080c60)